config/deployments/flagd/deployment.yaml

```diff
@@ -29,13 +29,13 @@ spec:
           readinessProbe:
             httpGet:
               path: /readyz
-              port: 8014
```

Comment:
please, adapt also the ZD tests manifest to use the new port :)

Reply:
@thisthat flagd-proxy uses the sync package from core, NOT the `flag-evaluation` package. The health-checks of flagd-proxy are not impacted by this change (which may or may not be good, since things will be a bit inconsistent.

Problem is, I'm not 100% sure I can add HTTP 1.1 health-check support to the sync package since it doesn't use `connect`.